modules/ROOT/pages/to-request-access.adoc

```diff
@@ -4,13 +4,27 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-APIs that have been protected by a Client ID enforcement policy require API consumers to create an application to hold the Client ID and Secret. This lets the app access the API as well as provide consumer analytics for an API developer.
+To implement an API, the client application must first request access. When the
+request is approved by the API, a contract is created between the client
+application and the API, and the client application is registered.
 
-If terms and conditions are set for an asset, you are prompted to accept the terms before access can be granted.
+APIs that are protected by a client ID enforcement policy require client
+applications to provide a client ID and client secret. The client ID and client
+secret credentials are automatically created when the client application is
+registered. These credentials enable the application to access the API and
```

Comment:
This is somewhat confusing since it seems to imply that only with this policy and this request analytics is enabled. Analytics is enabled for all tracked applications. In that case, they are generated even if no policy is applied.

Reply:
Fixed.